lib/jira/api.rb

```diff
@@ -62,7 +62,17 @@ def endpoint
     end
 
     def headers
-      { 'Content-Type' => 'application/json' }
+      args = {
+        'Content-Type' => 'application/json'
+      }
```

Comment:
if we return `{}` from `Jira::Core.cookie` instead of `nil`, we can get rid of the `.nil?` check